tools/metrics/main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"time"
+)
+
+func regressions() error {
+	qc := newDefaultQueryClient()
+	start := time.Now()
+	weeks := 7
+	srr, err := queryRegressions(qc, time.Now(), Weekly, weeks)
+	elapsed := time.Since(start)
+	fmt.Printf("Query took %v\n", elapsed)
+	if err != nil {
+		return err
+	}
+	for slot, sr := range srr {
+		if slot == 0 {
+			fmt.Printf("%s -> %s this week:", sr.interval.start.Format(DayFormat), sr.interval.end.Format(DayFormat))
```

Comment:
suggestion: leave rendering to the caller, and the logic for string construction here. Formally, change the signature to:
```
func regressions() (string, error) {
 ...
   return fmt.Sprintf("..."), nil
...
```

and at the caller side you'd `fmt.Fprintf(os.Stdout, regressions)` to denote the fact that we're printing regression strings consistently on stdout, not stderr.